File: 0392-is-subsequence/0392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        l1=0;
        l2=0;
        if not s:
            return True;
        for l2 in range(len(t)):
            if l1<len(s) and t[l2] == s[l1]:
                l1= l1+1;
        return True if l1 == len(s) else False;

File: 0392-is-subsequence/0392-is-subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int l1=0;
        if(s.length() == 0){
            return true;
        }
        for (int l2=0; l2<t.length(); l2++){
            if (l1<s.length() && t.charAt(l2) == s.charAt(l1)){
                l1=l1+1;
            }
        }
        if(l1==s.length()){
            return true;
        }
        return false;
    }
}